### PR TITLE
[19461] Fix Fast DDS nightly jobs

### PR DESCRIPTION
**Pull request by JLBuenoLopez-eProsima**
## Description

After merging eProsima/Fast-CDR#156 which includes an API break upgrading Fast CDR to v2.0.0, Fast DDS nightly workflows are failing. Until #3828 is merged, this is going to keep happening. This PR modifies the Fast CDR dependency fixing it to 1.1.x branch instead of master.

## Contributor Checklist
- [x] Commit messages follow the project guidelines. 
- [N/A] The code follows the style guidelines of this project. 
- [N/A] Tests that thoroughly check the new feature have been added/Regression tests checking the bug and its fix have been added; the added tests pass locally 
- [N/A] Any new/modified methods have been properly documented using Doxygen. 
- [N/A] Changes are ABI compatible. 
- [N/A] Changes are API compatible. 
- [N/A] New feature has been added to the `versions.md` file (if applicable).
- [N/A] New feature has been documented/Current behavior is correctly described in the documentation. 
    
- [N/A] Applicable backports have been included in the description.

## Reviewer Checklist
- [x] The PR has a milestone assigned.
- [x] Check contributor checklist is correct.
- [x] Check CI results: changes do not issue any warning.
- [x] Check CI results: failing tests are unrelated with the changes.

### CODE REVIEW

**Review by EduPonz — APPROVED** [review]

**Comment by JLBuenoLopez-eProsima** [comment]
Jenkins jobs must be updated to use CMake 3.22 for the CI to run.

File: .github/workflows/reusable-windows-ci.yml [hunk 1]
```diff
@@ -45,7 +45,7 @@ jobs:
       - name: Get minimum supported version of CMake
         uses: lukka/get-cmake@latest
         with:
-          cmakeVersion: '3.16.3'
+          cmakeVersion: '3.22.1'
```
Comment (richiware) [inline .github/workflows/reusable-windows-ci.yml:48]:
```suggestion
          cmakeVersion: '3.22.6'
```

**Review by richiware — DISMISSED** [review]

File: .github/workflows/reusable-windows-ci.yml (hunk 1 above)
Comment (JLBuenoLopez-eProsima) [reply .github/workflows/reusable-windows-ci.yml]: As it has been discussed internally, this PR builds Fast CDR v1.1.x so it is not required to upgrade the CMake. I have dropped the commit.

**Comment by JLBuenoLopez-eProsima** [comment]
@richiprosima pleae test this

**Review by EduPonz — DISMISSED** [review]

**Comment by JLBuenoLopez-eProsima** [comment]
@richiprosima please test this

**Comment by JLBuenoLopez-eProsima** [comment]
@richiprosima please test linux

**Comment by JLBuenoLopez-eProsima** [comment]
@richiprosima please test this

**Comment by JLBuenoLopez-eProsima** [comment]
@richiprosima please test this

**Comment by JLBuenoLopez-eProsima** [comment]
@richiprosima please test this

**Review by EduPonz — DISMISSED** [review]

**Comment by EduPonz** [comment]
@richiprosima please test this

**Comment by EduPonz** [comment]
@richiprosima please test mac

**Review by EduPonz — APPROVED** [review]

**Pull request merged**